pages/admin/request/create-renewal.tsx

```diff
@@ -305,9 +322,12 @@ export default function CreateRenewal() {
                   {`Permit Holder's Information`}
                 </Text>
                 <PermitHolderInformationForm
-                  type="RENEWAL"
-                  permitHolderInformation={permitHolderInformation}
-                  onChange={setPermitHolderInformation}
+                  permitHolderInformation={{ type: 'RENEWAL', ...permitHolderInformation }}
```

Comment:
```suggestion
                  permitHolderInformation={{ ...permitHolderInformation, type: 'RENEWAL', }}
```
Constructing an object like this takes place in the order of the fields being specified. Moving the type to the end will help us avoid all bugs that could (rarely) arise due to the type being incorrect, as we are hardcoding this value here and it should definitely be of type 'RENEWAL'